native/src/components/PageDetail.tsx

```diff
@@ -33,13 +39,25 @@ type PropsType = {
 
 const PageDetail: React.FC<PropsType> = (props: PropsType): ReactElement => {
   const { identifier, information, theme, language, onLinkClick, linkLabel } = props
+  const { t } = useTranslation('pois')
+
+  const navigationUrl = Platform.select({
+    ios: `maps:0,0?q=${information}`,
+    android: `geo:0,0?q=${information}`
+  })
   return (
     <>
       <DetailContainer theme={theme} language={language}>
         <Identifier theme={theme}>{identifier}: </Identifier>
         {information}
       </DetailContainer>
       {linkLabel && onLinkClick && <Button title={linkLabel} onPress={() => onLinkClick()} />}
+      {linkLabel && (
+        <>
+          <Spacer />
+          <Button title={t('navigation')} onPress={() => navigationUrl && Linking.openURL(navigationUrl)} />
```

Comment:
Do we need to check for `navigationUrl` here?

Reply:
yes since the platform.select could return string | undefined